### PR TITLE
refactor(app): align the wifi search screen with the hi-fi design

### PR DESCRIPTION
**Pull request by koji**
# Overview
design
[searching for network]
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen-Designs?type=design&node-id=7044-89795&t=ubkbm2Ao2X6xFzxI-0

[networks found]
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen-Designs?type=design&node-id=7261-96129&t=ubkbm2Ao2X6xFzxI-0

# Test Plan

# Changelog
- update components that are related to searching wifi screen
- update the above test cases

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/assets/localization/en/device_settings.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "characters_max": "17 characters max",
   "check_for_updates": "Check for updates",
   "checking_for_updates": "Checking for updates",
+  "choose_a_network": "Choose a network",
```
Comment (ewagoner) [inline app/src/assets/localization/en/device_settings.json]: I know the designs have both of these, but I'd push back on that. There's no reason i can see to have both.

File: app/src/App/OnDeviceDisplayApp.tsx [hunk 2]
```diff
@@ -241,7 +241,7 @@ export const OnDeviceDisplayApp = (): JSX.Element => {
   }, [dispatch, isIdle, usersBrightness])
 
   return (
-    <ApiHostProvider hostname="localhost">
+    <ApiHostProvider hostname="192.168.0.107">
```
Comment (ewagoner) [inline app/src/App/OnDeviceDisplayApp.tsx]: What's this address?

**Comment by ewagoner** [comment]
This doesn't seem to be refreshing the network list every five seconds. Is that still a to-do item?

File: app/src/App/OnDeviceDisplayApp.tsx (hunk 2 above)
Comment (koji) [reply app/src/App/OnDeviceDisplayApp.tsx]: oops, used for debugging

File: app/src/organisms/NetworkSettings/DisplayWifiList.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          </Flex>
+        </Btn>
+      </Flex>
+      <Flex justifyContent={JUSTIFY_CENTER} flex="2">
+        <StyledText as="h2" fontWeight={TYPOGRAPHY.fontWeightBold}>
+          {hasSsid ? t('choose_a_network') : t('select_a_network')}
+        </StyledText>
+      </Flex>
+      <Flex justifyContent={JUSTIFY_END} flex="1">
+        {isSearching && !hasSsid ? (
```
Comment (ewagoner) [inline app/src/organisms/NetworkSettings/DisplayWifiList.tsx]: I think we should show this spinner even if we have a ssid to visually show that it is refreshing every five seconds

**Comment by koji** [comment]
> This doesn't seem to be refreshing the network list every five seconds. Is that still a to-do item?

oh good, catch I didn't notice that update.
for mid-fi, we didn't do that to avoid keep updating the list.
I will update the component.

File: app/src/assets/localization/en/device_settings.json (hunk 1 above)
Comment (koji) [reply app/src/assets/localization/en/device_settings.json]: you mean `choose a network` and `select a network`?

Comment (ewagoner) [reply app/src/assets/localization/en/device_settings.json]: Yeah. That screams to me of an oversight -- I don't see any reason for the word to change one screen to the other.

Comment (koji) [reply app/src/assets/localization/en/device_settings.json:27]: okay let me ask about that to Mel

**Comment by koji** [comment]
- [x] solve conflicts
- [x] update the functions

**Comment by koji** [comment]
> This doesn't seem to be refreshing the network list every five seconds. Is that still a to-do item?

I just noticed that now we can do that by passing `interval`(number) to `useWifiList`.

File: app/src/organisms/NetworkSettings/DisplayWifiList.tsx (hunk 3 above)
Comment (koji) [reply app/src/organisms/NetworkSettings/DisplayWifiList.tsx]: Talked about this with Mel and fetching list every 5 sec and displaying spinner look a little bit weird.
So this part will be tweaked when she is in the office.

File: app/src/assets/localization/en/device_settings.json (hunk 1 above)
Comment (ecormany) [reply app/src/assets/localization/en/device_settings.json]: Good catch @ewagoner, I'd say `Select` for both. It's parallel with `Select a security type` in the same setup flow.

Comment (koji) [reply app/src/assets/localization/en/device_settings.json]: talked about this with Ed and Mel in person.
we decided to remove choose a network.
I will update.

thanks @ewagoner

**Comment by koji** [comment]
Close this PR since https://github.com/Opentrons/opentrons/pull/12799 covers this PR.

**Pull request closed without merging**